server/controllers/stub-data.ts

```diff
@@ -14,6 +14,7 @@ const STUB_ALLOWED = config.get('allowStubData') === 'true';
 const STUB_RESPONSES: StubResponse[] = [
   // include stubs here like below
   // ...eventMappings,
+  // ...transformedMedia,
```

Comment:
Is this not stubbed?

Reply:
Addressed in https://github.com/hmcts/darts-portal/pull/1118